dcat/rdf/dcat.ttl

```diff
@@ -85,9 +85,9 @@ Některé položky těchto slovníků nejsou zcela vhodné pro datové sady nebo
   skos:changeNote "Vlastnost v tomto kontextu byla přidána ve verzi DCAT 2.0"@cs ;
 .
 dct:PeriodOfTime
-  skos:definition "An interval of time that is named or defined by its start and end."@en ;
-  skos:definition "Un interválo de tiempo que se nombra o define por su principio y fin."@es ;
-  skos:definition "Časový interval který je pojmenovaný, nebo určený svým začátkem a koncem."@cs ;
+  skos:scopeNote "An interval of time that is named or defined by its start and end."@en ;
```

Comment:
Same as DCMI definition except for omitting "dates" at the end - http://dublincore.org/specifications/dublin-core/dcmi-terms/2012-06-14/?v=terms#PeriodOfTime

Reply:
I was unsure how to deal with these cases, there are other cases alike, i.e. dct:Location, dct:conformsTo. 
Are you suggesting to keep it as skos:definition?  I think we should keep the translation produced for them.

Reply:
In my opinion the DCMI definition is unnecessarily restrictive - a period-of-time may have the same date at the beginning and end!